test/definition/ApiDefinitionFactorySpec.scala

```diff
@@ -48,26 +47,8 @@ class ApiDefinitionFactorySpec extends UnitSpec with MockAppConfig {
         MockedAppConfig.endpointsEnabled(Version1) returns true
         MockedAppConfig.confidenceLevelCheckEnabled.returns(confidenceLevelConfig).anyNumberOfTimes()
```

Comment:
Doesn't depend on configdence level so this isn't needed.

In fact it looks like the testDefinitionWithConfidence method body can be the basis for a single test (no poiint in calling twice with different confidence levels any more).